app/src/main/java/com/kamron/pogoiv/activities/OcrCalibrationResultActivity.java

```diff
@@ -126,6 +126,10 @@ public static void startCalibration(@NonNull Context context,
     }
 
 
+    private OcrCalibrationResultActivity getOuter() {
```

Comment:
This is not needed, in the next comment I'll show how to solve this problem without defining a new function